prime-router/src/main/kotlin/azure/observability/event/ReportReceivedEvent.kt

```diff
@@ -0,0 +1,12 @@
+package gov.cdc.prime.router.azure.observability.event
+
+import gov.cdc.prime.router.ReportId
+import gov.cdc.prime.router.Topic
+
+data class ReportReceivedEvent(
+    val reportId: ReportId,
+    val topic: Topic,
+    val format: String,
+    val senderName: String,
+    val payload: String?,
```

Comment:
Hmm, wondering if this might be crazy big and not good to put in the events?  I'm thinking like a 1000+ ndjson FHIR bundle

Reply:
changed to payloadName